LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -31,6 +31,12 @@ class LockboxXCUITests: BaseTestCase {
         super.setUp()
     }
 
+    override func tearDown() {
+        navigator.goto(Screen.LockboxMainPage)
```

Comment:
The `navigator` should know where it is already. You don't need to add _just in case_ `goto`s.